File: Java/04102023 Array/LabTask/Factorial.java
import java.util.Scanner;

public class Factorial {
    int ans = 0;
    public int fact(int n){
        if(n==0)
            return 1;
        else
        return n * fact(n-1);
    }
    public static void main(String[] args) {
        System.out.println("Enter the number to check the factorial of a number: ");
        Factorial obj = new Factorial();
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();
        System.out.println(obj.fact(num));;
    }
    
}

File: C/seventhDay/factorial.cpp
// Factorial with return with arguments
#include <iostream>
using namespace std;

int fact(int num) {
    int ans = 1;
    for (int i = 1; i <= num; i++) {
        ans = ans * i;
    }
    return ans;
}

int main() {
    int n;
    cin >> n;
    cout << fact(n);
    return 0;
}